.vale/styles/RedHat/DoNotUseTerms.yml

```diff
@@ -4,6 +4,7 @@ ignorecase: true
 level: warning
 link: https://redhat-documentation.github.io/vale-at-red-hat/docs/main/reference-guide/donotuse/
 message: "%s"
+scope: raw
```

Comment:
@aireilly I don't know why but this rule wouldn't work without `scope: raw` 
At https://github.com/errata-ai/vale/issues/233#issuecomment-693607063 it says that regex is independent of the `raw` scope. ¯\_(ツ)_/¯

Reply:
I'll have a look.

Reply:
Ah. Is it because Vale works on the HTML, and when we generate HTML we get 
```
![image](https://github.com/redhat-documentation/vale-at-red-hat/assets/74046732/139045b5-9341-4658-819a-072e25f7855d)
```

Reply:
I would be inclined to remove the entire "Red" element from the rule if we can't make it work without scope.

Either that, or remove `ignorecase: true` and `scope: raw` and go with:

```
  \bred\b(?!-? *hat)|orange|yellow|green|blue|indigo|violet: 'Do not use colors to describe something unless it is also described non-visually, for example, with the name of the item.'
```

Reply:
:shrug:

Reply:
Setting "raw" also means that we would start linting for all these words in all text in an AsciiDoc file, incl inside codeblocks, and other mark up. Not ideal.